.github/workflows/main.yml

```diff
@@ -142,10 +147,6 @@ jobs:
           create-args: >-
             mamba
             python=${{ matrix.python-version }}
-      - name: Downgrade intake-esm
-        if: matrix.python-version == '3.9'
-        run: |
-          micromamba install -y -c conda-forge intake-esm=2023.11.10
```

Comment:
This is needed for the new `intake-esm`

Reply:
woups sorry I didn't check the git revert carefully!